js/stdlib/ts/ALGO.ts

```diff
@@ -1992,13 +2021,17 @@ export const connectAccount = async (networkAccount: NetworkAccount): Promise<Ac
             });
             const vres = await decode(vi, vvs, args);
```

Comment:
The thing we REALLY want to cache is the call to `getState_`

Reply:
And since we want to cache it, I think you should move the cache right next to the definition of `getState_`